src/app/main/component/eco-news/components/eco-news-detail/eco-news-detail.component.html

```diff
@@ -11,11 +11,16 @@
           </div>
         </a>
       </div>
-      <a [routerLink]="['/news/create-news']" [queryParams]="{ id: newsItem.id }">
-        <div class="edit-news" *ngIf="newsItem.author.id === userId">
-          {{ 'eco-news-single-view.edit-news-button' | translate }}
-        </div>
-      </a>
+      <div class="editDeleteBlock" *ngIf="newsItem.author.id === userId">
```

Comment:
would better to use kebab-case as in other classes

Reply:
Ok, I will change it to 'edit-delete-block'.